src/js/view/components/LogViewer.js

```diff
@@ -109,6 +138,10 @@ const LogViewer = props => {
     };
   }, []);
 
+  useEffect(() => {
+    setScrollPosition(minScrollPositionValue);
+  }, [props.meanByteValuesOfInitialLines[props.source.path]]);
+
```

Comment:
The culprit of the empty logviewer tab seems to be this effect. It sets the scrollposition and triggers another effect which tries to fetch new lines with values that has not had the time to be set to anything helpful at that time in the process.